packages/fbjs/src/datatransfer/DataTransfer.js

```diff
@@ -75,7 +75,7 @@ class DataTransfer {
   getText() {
     var text;
     if (this.data.getData) {
-      if (!this.types.length) {
+      if (!this.types.length || this.types.indexOf('Text' !== -1)) {
```

Comment:
@mindlink it looks like your parenthesis is at the wrong position...
`this.types.indexOf('Text') !== -1` is probably that what you actually mean?